src/NHSD.BuyingCatalogue.Identity.Api/Controllers/AccountController.cs

```diff
@@ -89,5 +89,30 @@ public IActionResult Registration()
         {
             return View();
         }
+
+        [HttpGet]
+        public IActionResult ForgotPassword()
+        {
+            return View();
+        }
+
+        [HttpPost]
+        public IActionResult ForgotPassword(ForgotPasswordViewModel viewModel)
```

Comment:
Do we not want a `[ValidateAntiForgeryToken]` here? (At least until we set it up as default).